lib/google_scraping/dashboard/keywords.ex

```diff
@@ -16,11 +16,16 @@ defmodule GoogleScraping.Dashboard.Keywords do
 
   ## Examples
 
-      iex> list_keywords(user_id)
+      iex> list_keywords(user_id, nil)
       [%Keyword{}, ...]
 
+      iex> list_keywords(user_id, %{"query" => "dog"})
+      [%Keyword{}, ...]
   """
-  def list_keywords(user_id), do: Repo.all(KeywordQuery.user_keywords_query(user_id))
+  def list_keywords(user_id, params \\ %{}) do
+    search_phrase = get_in(params, ["query"])
+    Repo.all(KeywordQuery.user_keywords(user_id, search_phrase))
```

Comment:
Another place where we can pipe vertically 😄 

```suggestion
    user_id
    |> KeywordQuery.user_keywords(search_phrase)
    |> Repo.all()
```